go/ct/st/memory.go

```diff
@@ -88,9 +88,16 @@ func (m *Memory) Grow(offset, size uint64) {
 // ExpansionCosts calculates the expansion costs for the given offset and size.
 // It does not grow memory. It also returns offset and size converted to uint64.
 func (m *Memory) ExpansionCosts(offset_u256, size_u256 U256) (memCost vm.Gas, offset, size uint64) {
-	if !size_u256.IsUint64() || (!offset_u256.IsUint64() && !size_u256.IsZero()) {
+	if !size_u256.IsUint64() {
```

Comment:
maybe we should define our coding style for this kind of cases.

Reply:
It is not only style, the logic is different, geth will ignore overshooting offsets if the size is 0, generating no expansion. Previous code did generate expansion for size 0.